content/examples/03-node/03-tools/index.mdx

```diff
@@ -22,8 +22,8 @@ The following sections will guide you through tool use with Node.js and the Verc
       href: '/examples/node/tools/call-tools-in-parallel',
     },
     {
-      title: 'Call Tools with Automatic Roundtrips',
-      href: '/examples/node/tools/call-tools-with-automatic-roundtrips',
+      title: 'Multi-step Tool Calls',
+      href: '/examples/node/tools/tool-calls-with-automatic-roundtrips',
```

Comment:
connected to previous point - check links are correct (doesn't seem in line with current name)